src/Build/Evaluation/Context/EvaluationContextBase.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//-----------------------------------------------------------------------
+// </copyright>
+
+using PublicEvaluationContext = Microsoft.Build.Framework.EvaluationContext.EvaluationContext;
```

Comment:
Changing the namespace of `EvaluationContext` would obviously remove the need for this